core/services/workflows/models_test.go

```diff
@@ -19,26 +19,30 @@ func TestParse_Graph(t *testing.T) {
 			name: "basic example",
 			yaml: `
 triggers:
-  - id: "a-trigger"
+  - id: "a-trigger@1"
+    config: {}
 
 actions:
-  - id: "an-action"
+  - id: "an-action@1"
+    config: {}
```

Comment:
IRL workflow scenarios seem to always have a config, hence why this is a required field rather than a nullable one. It does look at little odd when we're using fixture workflows though

Reply:
> It does look at little odd when we're using fixture workflows though

Could this fixture be made more realistic, then?